src/Constants.js

```diff
@@ -151,11 +152,28 @@ exports.GiveawaysManagerOptions = {
     }
 };
 
+/**
+ * The options for a forced ending or reroll of a giveaway.
+ * The manager attempts to get "giveaway.messages" and "giveaway.embedColor|giveaway.embedColorEnd" (depending on "end or reroll"), from the giveaway that was started closest to the giveaway that is forced (in the same server).
+ * @typedef GiveawayForceOptions
+ *
+ * @property {boolean} [force=false] If the ending or reroll should be forced, when the giveaway is not found.
+ * @property {Discord.TextChannel|Discord.NewsChannel|Discord.ThreadChannel} channel The channel in which the giveaway that is forced is.
+ * @property {boolean} [saveGiveawayInDatabase=true] If the giveaway that is forced should be saved in the runtime giveaway array and the database.
+ * @property {number} [closestGiveawayThreshold=86400000] How big the "giveaway.startAt" difference to the closest giveaway can be, before the closest giveaway is considered invalid. Default is 1 day. If there is no valid closest giveaway then "giveaway.messages" and "giveaway.embedColor|giveaway.embedColorEnd" will have default values (Set this property to "0" to always use default messages). 
+ */
+exports.GiveawayForceOptions = {
+    force: false,
+    channel: null,
+    saveGiveawayInDatabase: true,
+    closestGiveawayThreshold: 86400000
```

Comment:
LMK your thoughts about this 1day default threshold.
I chose this cause giveaways are not all to frequent.
But you could say since thats the case, we could also make the default even bigger?
Or you guys think this default is fine?